src/core/CCOpt.ml

```diff
@@ -54,10 +58,16 @@ let (<*>) f x = match f, x with
 
 let (<$>) = map
 
-let (<+>) a b = match a with
+let or_ b a = match a with
```

Comment:
Possible, but this must be emphasized in the documentation, or perhaps even with a labelled argument (e.g. `a |> or_ ~else_:b |> or_ ~else_:c`). What do you think?